src/placeos-rest-api/controllers/flux.cr

```diff
@@ -0,0 +1,37 @@
+require "./application"
+
+module PlaceOS::Api
+  class Flux < Application
+    include Utils::CoreHelper
+
+    # use influxdb path for any existing influx clients
+    base "/api/v2"
```

Comment:
```suggestion
    base "/api/engine/v2/influx/"
```

Could we scope this to be more indicative that it's backed by influx, or at least more specifically related to time-series queries

NOTE: `/api/v2` is the incorrect base by itself

Reply:
Oh I see, this is a straight-up proxy.
Could potentially get messy?
In the future, i think we should have a dedicated service proxy to remove this from the API